templates/cisco_ios_show_interfaces_switchport.textfsm

```diff
@@ -20,6 +20,7 @@ Start
   ^\s*Trunking\s+Native\s+Mode\s+VLAN:\s+${NATIVE_VLAN}
   ^\s*Voice\s+VLAN:\s+${VOICE_VLAN}
   ^\s*Trunking\s+VLANs\s+Enabled:\s+${TRUNKING_VLANS}$$
+  ^\s+${TRUNKING_VLANS}$$
```

Comment:
```suggestion
  ^\s*Trunking\s+VLANs\s+Enabled:\s+${TRUNKING_VLANS},\s*$$ -> Trunk
  ^\s*Trunking\s+VLANs\s+Enabled:\s+${TRUNKING_VLANS}\s*$$
  
Trunk
  ^\s+${TRUNKING_VLANS},\s*$$ -> Start
  ^\s+${TRUNKING_VLANS}\s*$$
```

Reply:
Implemented your exact suggestions and worked as intended!